src/libraries/System.Private.CoreLib/src/System/Number.NumberToFloatingPointBits.cs

```diff
@@ -113,6 +148,665 @@ public FloatingPointInfo(ushort denormalMantissaBits, ushort exponentBits, int m
             1e22,   // 10^22
         };
 
+        private static ReadOnlySpan<ulong> s_Pow5128Table => new ulong[] {
```

Comment:
In the original, this was...
```
    private readonly static ulong[] power_of_five_128 = {
```


See https://github.com/CarlVerret/csFastFloat/blob/master/csFastFloat/Constants/Constants.cs

The way it is written here looks like a heap allocation.

Reply:
@CarlVerret can you change it back to `static ulong[] power_of_five_128` as it's not clear when exactly we'll have Roslyn support for `ReadOnlySpan<ulong> => new ulong`.

@lemire this pattern will be optimized by Roslyn compiler, it's just not clear when exactly, see discussion here https://github.com/dotnet/runtime/pull/62301#discussion_r761755401